src/Slides.js

```diff
@@ -0,0 +1,38 @@
+import { URL } from 'url';
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+
+  /*
+    INVALID:
```

Comment:
This comment can be deleted,  since these should be reflected in the tests.

Reply:
ok, i will delete that, i just put theme here just to check them easily.